sasmodels/models/lib/gauss20.c

```diff
@@ -6,6 +6,14 @@
  *  Copyright 2007 __MyCompanyName__. All rights reserved.
  *
  */
+ #ifdef GAUSS_N
```

Comment:
As far as I can tell, gauss 20 is never called. Should we remove it?

Reply:
I opted to leave it since it may be used in some plugin models.

Reply:
Possibly yes remove it.  I replaced Gauss20 in a couple of models with Gauss76 ages ago when converting them. However will there be any potential backwards compatibility issues with the model marketplace etc? (Or would a Gauss20 be auto generated if used?)

Reply:
I don't want to rely on this working for arbitrary installs.   My hack writes the generated gauss file to builtin model path, but the user may not have write-access to this directory if the packages is installed for all users.  This will need to be fixed if we make the integration loop size a user-controlled parameter from the GUI.